blocks/cards/cards.css

```diff
@@ -0,0 +1,81 @@
+.block.cards {
+  display: grid;
+  grid-template-columns: repeat(3, minmax(0, 1fr));
+  grid-auto-rows: 1fr;
+  gap: 1rem;
+}
+
+.block.cards .card .url {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  top: 0;
+  left: 0;
+  padding: 0;
+  opacity: 0;
+  margin: 0;
+}
+
+.block.cards .card .url > a {
+  height: 100%;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.block.cards .card {
+  position: relative;
+  background-color: white;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  text-align: center;
+  padding: 0.75rem;
+  font-size: 1rem;
+  border-radius: 16px;
+}
```

Comment:
You might need to add `justify-content: center` for viewport <900px as otherwise, it is introducing unrequired empty space as below:

![Screenshot 2024-07-19 at 17 15 30](https://github.com/user-attachments/assets/1be6457a-0ecb-464f-bfef-d0ffd60249e3)

Reply:
No, there is no unrequired empty space. That space is used to strech the card divs to the biggest size (last card in your example) which has a bigger height than the one hovered. 
![image](https://github.com/user-attachments/assets/abeae4f7-a3b2-4fe5-9884-0d3103d9f394)
![image](https://github.com/user-attachments/assets/5f91ea01-3bf3-4af8-8d53-f30fd8a4de21)